checker/src/main/java/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -130,10 +130,9 @@ protected boolean commonAssignmentCheck(
         if (TreeUtils.isFieldAccess(varTree)) {
             // cast is safe: a field access can only be an IdentifierTree or MemberSelectTree
             ExpressionTree lhs = (ExpressionTree) varTree;
-            ExpressionTree y = valueExp;
             VariableElement el = TreeUtils.variableElementFromUse(lhs);
             AnnotatedTypeMirror xType = atypeFactory.getReceiverType(lhs);
-            AnnotatedTypeMirror yType = atypeFactory.getAnnotatedType(y);
+            AnnotatedTypeMirror yType = atypeFactory.getAnnotatedType(valueExp);
```

Comment:
The name `yType` is now a bit unmotivated.
Can you rename both `xType` and `yType` to make this more meaningful? Maybe `lhsType` and `valueExpType`? Other ideas?

Reply:
@wmdietl The name make sense to me. However, I am wondering why in this method the argument is `valueExp` and in super method it is `valueExpTree`.

Reply:
Sure, make that consistent too.

Reply:
Done.